src/group_theory/subgroup.lean

```diff
@@ -963,8 +1000,8 @@ instance center_normal : (center G).normal :=
 end⟩
 
 variables {G} (H)
-/-- The `normalizer` of `H` is the largest subgroup of `G` inside which `H` is normal. -/
-@[to_additive "The `normalizer` of `H` is the largest subgroup of `G` inside which `H` is normal."]
+/-- The `normalizer` of `H` is the smallest subgroup of `G` inside which `H` is normal. -/
+@[to_additive "The `normalizer` of `H` is the smallest subgroup of `G` inside which `H` is normal."]
```

Comment:
Are you sure about this? Isn't `H` the smallest subgroup of `G` inside which `H` is normal?

Reply:
The smallest subgroup of `G` that contains `H` and inside which `H` is normal is `H` itself. So this change of the docstring should be undone.
At the same time, I don't think this "universal property" of the normalizer is in Lean. We should add it.